From 79b769c42f096616bb732671060b349df07eec74 Mon Sep 17 00:00:00 2001
From: Robert Yang <liezhi.yang@windriver.com>
Date: Mon, 12 May 2014 15:12:41 +0800
Subject: [PATCH] anaconda: don't destroy all devices by default

Once anaconda found a unknown disk, it would prompt whether
re-initialize the disk, this is OK, but there is a button:

Apply to all:
"Apply my choice to all devices with undetected partitions"

This is a dangerous action, we need default it to "no" rather than "yes".

Upstream-Status: Pending

Signed-off-by: Robert Yang <liezhi.yang@windriver.com>
---
 data/ui/reinitialize-dialog.glade | 2 +-
 pyanaconda/text.py                | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/data/ui/reinitialize-dialog.glade b/data/ui/reinitialize-dialog.glade
index 26975b4..967e84e 100644
--- a/data/ui/reinitialize-dialog.glade
+++ b/data/ui/reinitialize-dialog.glade
@@ -222,7 +222,7 @@ Are you sure this device does not contain valuable data?</property>
 		  <property name="use_underline">True</property>
 		  <property name="relief">GTK_RELIEF_NORMAL</property>
 		  <property name="focus_on_click">True</property>
-		  <property name="active">True</property>
+		  <property name="active">False</property>
 		  <property name="inconsistent">False</property>
 		  <property name="draw_indicator">True</property>
 		</widget>
diff --git a/pyanaconda/text.py b/pyanaconda/text.py
index 6145699..6396f25 100644
--- a/pyanaconda/text.py
+++ b/pyanaconda/text.py
@@ -259,7 +259,7 @@ class InstallInterface(InstallInterfaceBase):
         text = TEXT_REINITIALIZE % {"description": description, "size": size, "devicePath": path}
         grid.add(TextboxReflowed(70, text), 0, 0)
 
-        all_devices_cb = Checkbox(TEXT_REINITIALIZE_ALL, isOn=True)
+        all_devices_cb = Checkbox(TEXT_REINITIALIZE_ALL, isOn=False)
         grid.add(all_devices_cb, 0, 1, padding=(0, 1, 0, 0))
 
         buttons = [(_("Yes, discard any data"), "yes"),
-- 
1.8.3.1

